magner/lib/views/components/table/cell.vue

```diff
@@ -116,9 +118,29 @@ export default defineComponent({
       }
 
       if (props.columnConfig.view.type === 'tags') {
+        const tags = [];
+        if (!Array.isArray(content)) {
+          tags.push({
+            text: content,
+            backgroundColor: '#ecf5ff',
+          });
+        } else {
+          content.map((item: unknown) => {
+            if (typeof item === 'object') {
+              tags.push(item);
+            } else {
+              tags.push({
+                text: item,
+                backgroundColor: '#ecf5ff',
+              });
+            }
+
+            return item;
+          });
+        }
```

Comment:
Praise: well done here! Nice job with handling tags. Also, this change adds a room for the future improvement!

Proposal (non-blocking, maybe for the future): extend the `ViewTags` interface by allowing users to configure the background and text colors for the tags. For example, look at the table column config API proposal:

```ts
{
  prop: 'type',
  label: 'User type',
  view: {
    type: 'tags',
    colorMap: {
      'admin': '#00F',
      'manager': '#0F0',
      'basic': '#F00',
    },
    backgroundColorMap: {
      'admin': '#0F0',
      'manager': '#F00',
      'basic': '#00F',
    },
  },
}
```